fala/templates/adviser/results.html

```diff
@@ -54,6 +54,18 @@ <h1 class="govuk-heading-xl">Search results</h1>
         }) }}
       </form>
     </div>
+
+    {% if data.count >= 1 %}
+      <div class="printing-button">
+        <div class="print-link__background govuk-!-display-none-print govuk-!-margin-top-3 govuk-!-margin-bottom-5">
+          <button id="printButton" class="govuk-link govuk-body-s print-link__button">
+            <img src="/static/images/icon-print.png" alt="Print Icon" class="print-link__icon">
```

Comment:
Why could you not have the icon as a background import? It's actually better if it is a background image.

See how HMRC and GOVUK do print in their service. They use CSS to apply the icon to the background. Makes it more browser friendly and better for performance (Though that is tiny). 

I imagine as well for accessibility a screen reader will also read out there is an image when in fact icons shouldn't really be read out by screen readers.

Reply:
Have adjusted.